### PR TITLE
Add annotations to templates

### PR DESCRIPTION
**Pull request by Thearas**
# Pull Request

## What does this PR do?

Implement #66 . I add `annotations` to pvc, svc and serviceAccount.

I saw that no one had done this issue and I just had the time😊. I am a beginner in Helm, if there is a mistake, please feel free to point out.

PTAL @eskombro

### CODE REVIEW

File: charts/meilisearch/templates/service.yaml [hunk 1]
```diff
@@ -10,6 +10,10 @@ metadata:
     {{- with .Values.customLabels }}
       {{- toYaml . | nindent 4 }}
     {{- end }}
+{{- with .Values.service.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}
```
Comment (eskombro) [inline charts/meilisearch/templates/service.yaml:16]:
```suggestion
  {{- with .Values.service.annotations }}
  annotations:
    {{ toYaml . | nindent 4 }}
  {{- end }}
```

File: charts/meilisearch/templates/serviceaccount.yaml [hunk 2]
```diff
@@ -10,3 +10,7 @@ metadata:
     {{- with .Values.customLabels }}
       {{- toYaml . | nindent 4 }}
     {{- end }}
+{{- with .Values.serviceAccount.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}
```
Comment (eskombro) [inline charts/meilisearch/templates/serviceaccount.yaml:16]:
```suggestion
  {{- with .Values.serviceAccount.annotations }}
  annotations:
    {{ toYaml . | nindent 4 }}
  {{- end }}
```

File: charts/meilisearch/values.yaml [hunk 3]
```diff
@@ -30,13 +30,18 @@ environment:
   # secret. Otherwise the below value of MEILI_MASTER_KEY will be used instead.
   # MEILI_MASTER_KEY:
 
+serviceAccount:
+  # Annotations to add to the service account
```
Comment (eskombro) [inline charts/meilisearch/values.yaml:34]:
```suggestion
```
IMO this elements name is self-explanatory, what do you think?

File: charts/meilisearch/README.md [hunk 4]
```diff
@@ -58,6 +58,10 @@ helm uninstall <your-service-name>
 | | |
 | `image.pullPolicy`               | MeiliSearch image pull policy                                  | `IfNotPresent`
 | | |
+| `serviceAccount.annotations`     | Additional annotations for service account                     | `{}`
+| | |
+| `podAnnotations`                 | Annotations for the pods                                       | `{}`
```
Comment (eskombro) [inline charts/meilisearch/README.md:63]: `podAnotations` wasn't added to the readme, thanks! 👌🏽

**Review by eskombro — COMMENTED** [review]
Hey @Thearas 

Thanks a lot for the contribution! I added a few comments and suggestions for the syntax, but I really appreciate your time to make it possible to add annotations for those resources! 

This repository is now maintained by @alallema so I let her do the approval process, my suggestions/review are therefore just comments :)

Thanks for contributing to MeiliSearch <3

File: charts/meilisearch/README.md [hunk 5]
```diff
@@ -72,6 +76,8 @@ helm uninstall <your-service-name>
 | | |
 | `service.type`                   | Kubernetes Service type                                        | `7700`
 | | |
+| `service.annotations`            | Additional annotations for service account                     | `{}`
```
[the remark this answers is not in the document]
Comment (eskombro) [reply charts/meilisearch/README.md:79]: service ~acccount~ :D

**Review by alallema — COMMENTED** [review]
Hi @Thearas,
Thanks for your contribution and Thanks, @eskombro for your review 😊.
You need to update the version in `charts/meilisearch/Chart.yaml` otherwise the github-action will be failed.
Don't hesitate to check the [contributing guide](https://github.com/meilisearch/meilisearch-kubernetes/blob/main/CONTRIBUTING.md).

**Comment by Thearas** [comment]
Thanks for your comments 😊.

PTAL again @alallema

**Review by alallema — APPROVED** [review]
Thanks a lot for the contribution @Thearas 
LGTM! 🔥

**Comment by alallema** [comment]
bors merge

**Pull request merged**